tests/worker_deployment_version_test.go

```diff
@@ -591,7 +591,50 @@ func (s *DeploymentVersionSuite) TestDeleteVersion_ValidDelete() {
 	}, time.Second*5, time.Millisecond*200)
 
 	// idempotency check: deleting the same version again should succeed
-	s.tryDeleteVersion(ctx, tv1, true)
+	s.tryDeleteVersion(ctx, tv1, true, false)
+}
+
+func (s *DeploymentVersionSuite) TestDeleteVersion_ValidDelete_SkipDrainage() {
+	s.T().Skip("skipping this test for now until I make TTL of pollerHistoryTTL configurable by dynamic config.")
```

Comment:
I un-skipped and ran this test locally with short pollerHistoryTTL and it passed